sos/report/plugins/crio.py

```diff
@@ -113,4 +125,13 @@ def _get_crio_goroutine_stacks(self):
         if result['status'] == 0:
             self.add_copy_spec("/tmp/crio-goroutine-stacks*.log")
 
+
+class CosCRIO(Plugin, CosPlugin):
+
+    def setup(self):
+        super(CosCRIO, self).setup()
+
+        self.add_journal(units='containerd')
```

Comment:
Are there other `containerd` related collections that'd be useful for COS? The `docker` plugin is what captures `/etc/containerd`, so I'm thinking it may be best to instead create a new `containerd` plugin at this point and put both of these collections there.

Reply:
That would be great but as of now, we would just need the `/etc/containerd`, container pods as well as the containerd service logs.